src/hid/hid-device.cpp

```diff
@@ -384,5 +430,12 @@ namespace librealsense
 
             return ep;
         }
+
+        void rs_hid_device::set_gyro_scale_factor(double scale_factor) 
```

Comment:
Who is calling `set_gyro_scale_factor`?
What happens until this is called? The old struct size will be used even though the new size is received from FW?

Reply:
d400-motion sets the gyro_scale_factor in ctor, after creating hid-device